Principal menu feature

Denobi opened the pull request.

Denobi committed:
fix: add main.java file

Denobi committed:
refactor: improve menu code, functions logic and variables name

Alisson-Lima commented:
**PRÓXIMOS PASSOS**

 Criar função reutilizável para os menus (printMenuOptionsOfSelectedMenu):
- Parâmetros (Nome do menu, opções do menu);
- Opções fixas (Retornar ao menu principal, sair da aplicação);

Aplicar função printMenuOptionsOfSelectedMenu em tudo.

++++++++++++++++++++++++++++++++++++++++

Seguir com login e cadastro.

Denobi committed:
refactor: create functions default and login create

Primeira versão do login e alteração no modo de criação dos menus.

Denobi committed:
Delete .gitignore

Denobi committed:
Update Login and Create User options

Atualização na tela de login e adição da opção para criar Usuário. Necessita de função para armazenar as informações!

Denobi committed:
atualizando main
